### DISCUSSION TITLE
Issue with plot count as plots are added

### DISCUSSION THREAD

**Discussion by cpereyra5199** [Chia-Network/chia-blockchain#7316] [Support]
Hello, I just updated to the new chia version and have started removing old plots and plotting the new pool plots.  New plots are being created successfully.  However, I am having to manually update plots from the plots view in order for the pool view to detect the new plots.  If I don’t update plots, the pool view doesn’t seem to pick up new plots as they come in and neither does the pool website tracking the plots.  Any idea what could be causing this issue?
![IMG_0002](https://user-images.githubusercontent.com/8915559/125146387-86500d00-e0f3-11eb-94fe-48791c2897fb.jpeg)
![IMG_0003](https://user-images.githubusercontent.com/8915559/125146388-8819d080-e0f3-11eb-9d4e-c70ff208e831.jpeg)

Thank you,

**Comment by Tao-ath** [comment]
I have the same problem

**Comment by mariano54** [comment]
Fixed in 1.2.1

**Reply by cpereyra5199** [reply]
Still running into this issue after upgrading to 1.2.1.  Pool plot counts are still not updating unless I manually refresh plots.

**Comment by cpereyra5199** [comment]
I am on 1.2.2 and still having this issue happen.  Is there any updates on when this will be resolved?

**Reply by cpereyra5199** [reply]
To clarify, it looks like the first plot is detected when starting the gui, however, after the first one the plot count does not update on the pools section.
